### DISCUSSION TITLE
Has anyone, not in hpool, won any xch the last month?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#6393] [General]
Has anyone, who is not in hpool, won any xch for the last 3-4 weeks?
Just wondering...

**Comment by XIII39** [comment]
I got a block on May 7th and another on on May 11th but... nothing since

**Comment by tydeno** [comment]
> Has anyone, who is not in hpool, won any xch for the last 3-4 weeks?
> Just wondering...

Like a week ago yes. At 400 Plots.

**Comment by paganig** [comment]
my first two xch like 23 hours ago (i have only about 70 plots, joined late april just for fun)

**Comment by 21497724** [comment]
I have two blocks yesterday

**Comment by westfruit** [comment]
i win  3~4 blocks  per day,i have 14pb plots and i not join other's pools

**Reply by tydeno** [reply]
Thats alot of HDDs for playing Bingo ;)
